tests/firedrake/test_0init.py

```diff
@@ -1,21 +1,8 @@
-import pytest
 import os
 from firedrake import *
 from pathlib import Path
 
 
-# See https://pytest-xdist.readthedocs.io/en/stable/how-to.html#identifying-the-worker-process-during-a-test
-@pytest.mark.skipif(
-    "PYTEST_XDIST_WORKER_COUNT" in os.environ.keys()
-    and int(os.environ["PYTEST_XDIST_WORKER_COUNT"]) > 1,
-    reason="Must be run first"
-)
-def test_pyop2_not_initialised():
-    """Check that PyOP2 has not been initialised yet.
-       The test fails if another test builds a firedrake object not in a fixture."""
-    assert not op2.initialised()
```

Comment:
I have removed this because it does not work with `pytest-split` and has questionable value anyway.